### PR TITLE
Added theme "Smoothie"

### PR DESCRIPTION
**Pull request by PapiPeppers**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description

Added new theme named "smoothie" to /themes. Preview images in forked repo's README.

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# Forked Repo Changes
```
Comment (JanDeDobbeleer) [inline README.md]: this will also appear in this repo so it needs to be removed

File: themes/smoothie.omp.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+	  "alignment": "right",
+	  "newline": false,
+	  "segments": [
+		{
+		  "type": "exit",
+		  "style": "plain",
+		  "foreground": "#fb0207",
+		  "properties": {
+			"prefix": "",
+			"postfix": " ",
```
Comment (JanDeDobbeleer) [inline themes/smoothie.omp.json]: default value, can be removed

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
You need to remove the images and the addition to the README. Other than that, just 1 remark 👍🏻

**Comment by PapiPeppers** [comment]
all should be good to go now. thanks!

**Comment by JanDeDobbeleer** [comment]
@all-contributors please add @PapiPeppers for design

**Pull request merged**